src/library_manager/lib_manager.c

```diff
@@ -25,6 +25,10 @@
 #include <zephyr/cache.h>
 #include <zephyr/drivers/mm/system_mm.h>
 
+#if CONFIG_LIBRARY_AUTH_SUPPORT
+#include <sof/auth/auth_api.h>
```

Comment:
So this would go to platform folder and each platform (that supports LIB auth) would need to provide one.

Reply:
fixed